frog/imports/client/Wiki/index.js

```diff
@@ -185,7 +187,8 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
     wikiStore.setPages(this.wikiDoc.data.pages);
     this.setState({
       pagesData: wikiStore.pages,
-      settings: this.wikiDoc.data.settings
+      settings: this.wikiDoc.data.settings,
```

Comment:
Make sure that `wikiDoc.data.settings` also has the type `WikiSettingsT` perhaps by moving the type definition to where `wikiDoc` is defined.

Reply:
`wikiDoc` is the shareDB document, we will need to add validation rules in MongoDB